index.html

```diff
@@ -116,12 +126,15 @@ <h3>CANOPY</h3>
                     <li>Javascript</li>
                 </ul>
                 <button class="See-project-button" type="button">See Project</button>
```

Comment:
- [ ] Please, make sure each button matches the Figma design.

Your design:
![image](https://user-images.githubusercontent.com/86069740/236354959-f4aa5381-a60b-4fdb-9e78-14a302574eb7.png)

Figma design:
![image](https://user-images.githubusercontent.com/86069740/236354997-00bcb7aa-446d-4d05-800e-d72e3109464e.png)